.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - checkout
       - run:
           name: Install requirements
-          command: pip install -r requirements.txt
+          command: pip install -r requirements.txt --process-dependency-links
```

Comment:
I think this pip option is being deprecated, we need it?

Reply:
We need it, or else this dependency from the openchemistry `setup.py` will not be satisfied
```python
    dependency_links=[
        'git+ssh://git@github.com/cclib/cclib.git#egg=cclib-1.6.1'
    ]
```